src/app/[lang]/about/page.tsx

```diff
@@ -1,29 +1,66 @@
 "use client";
+
+import AboutSection from "@/components/about-section";
 import AboutSectionLayout from "@/components/about-section-layout";
+import AppLink from "@/components/link";
 import ProjectCard from "@/components/project-card";
 import { useDictionary } from "@/contexts/dictionary-provider";
 
 export default function About() {
   const dict = useDictionary();
+
   return (
-    <main>
-      <AboutSectionLayout
-        linkName="see_more"
-        titleOrientation="vertical"
-        title={dict.about.projects.title}
-        subtitle={dict.about.projects.description}
-        href="/projects"
-        overflows
-      >
-        <div
-          className="flex flex-col md:flex-row md:gap-4"
-          style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
-        >
-          <ProjectCard type="sei" />
-          <ProjectCard type="bugsbyte" />
-          <ProjectCard type="coderdojo" />
+    <>
+      <div className="flex flex-col">
+        <div className="flex flex-col bg-white">
+          <AboutSectionLayout
+            linkName="see_more"
+            titleOrientation="vertical"
+            title={dict.about.projects.title}
+            subtitle={dict.about.projects.description}
+            href="/projects"
+          >
+            <div
+              className="flex flex-col md:flex-row md:gap-4"
+              style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
+            >
+              <ProjectCard type="sei" />
+              <ProjectCard type="bugsbyte" />
+              <ProjectCard type="coderdojo" />
+            </div>
+          </AboutSectionLayout>
         </div>
-      </AboutSectionLayout>
-    </main>
+        <AboutSection>
+          <div className="border-t border-black/10 pt-10">
```

Comment:
We don't need this border since the component above (AboutSectionLayout) already applies a bottom border so that we can "stack" sections and automatically have the borders between them. With this border you have, we can notice a "double border" if you look close.

As I meantioned in other comments, we should use gaps here to separate the elements inside instead of relying on individual margins or paddings on each child element. This way, if we want to change the gap size, we just change one value.

```suggestion
          <div className="flex flex-col gap-5 py-10 md:py-16">
```